aerospike/pyproject.toml

```diff
@@ -41,7 +41,7 @@ text = "BSD-3-Clause"
 [project.optional-dependencies]
 deps = [
     "aerospike==4.0.0; sys_platform != 'win32' and sys_platform != 'darwin' and python_version < '3.0'",
-    "aerospike==7.1.1; sys_platform != 'win32' and sys_platform != 'darwin' and python_version > '3.0'",
+    "aerospike==8.0.0; sys_platform != 'win32' and sys_platform != 'darwin' and python_version > '3.0'",
```

Comment:
It broke the agent build: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/202606630

It's too late to update the buildimages so could you also exclude this change and add a comment?

Reply:
done!